.github/renovate.json

```diff
@@ -8,6 +8,7 @@
     "branchTopic": "lock-file-maintenance-{{packageFile}}",
     "schedule": ["after 4am and before 5am on monday"]
   },
+  "baseBranches": ["main", "3.0.x"],
```

Comment:
I hope this will not reopen all the closed/merged PR we have in main.

Reply:
It probably will create new ones - because they don't exist in 3.0.x